File: hexlet/programming_introduction_js/lesson_9.js
const sequenceSum = (begin, end) => {
  // BEGIN (write your solution here)
  let acc = 0;

  if (begin > end) {
    return NaN;
  }
  if (begin === end) {
    return begin;
  }

  return begin = begin + sequenceSum(begin + 1, end);
  // END
};

File: hexlet/ruby_introduction/lesson_9.rb
def result(a, b, c, d)
  # BEGIN (write your solution here)
  if a === c && b === d
    return 2
  elsif  a > b && c > d or a < b && c < d or a === b && c === d
    return 1
  else
    return 0
  end
  # END
end
